source/docs/software/commandbased/profile-subsystems-commands.rst

```diff
@@ -27,65 +19,7 @@ When subclassing ``TrapezoidProfileSubsystem``, users must override a single abs
-
-#### Constructor Parameters
-
-Users must pass in a set of ``TrapezoidProfile.Constraints`` to the ``TrapezoidProfileSubsystem`` base class through the superclass constructor call of their subclass.  This serves to constrain the automatically-generated profiles to a given maximum velocity and acceleration.
-
-Users must also pass in an initial position for the mechanism.
-
-Advanced users may pass in an alternate value for the loop period, if a non-standard main loop period is being used.
-
-### Using a TrapezoidProfileSubsystem
-
-Once an instance of a ``TrapezoidProfileSubsystem`` subclass has been created, it can be used by commands through the following methods:
-
-#### setGoal()
-
-.. note:: If you wish to set the goal to a simple distance with an implicit target velocity of zero, an overload of ``setGoal()`` exists that takes a single distance value, rather than a full motion profile state.
-
-The ``setGoal()`` method can be used to set the goal state of the ``TrapezoidProfileSubsystem``.  The subsystem will automatically execute a profile to the goal, passing the current state at each iteration to the provided `useState()`_ method.
-
-.. tab-set-code::
-
-   .. code-block:: java
-
-      // The subsystem will execute a profile to a position of 5 and a velocity of 3.
-      examplePIDSubsystem.setGoal(new TrapezoidProfile.State(5, 3);
-
-   .. code-block:: c++
-
-      // The subsystem will execute a profile to a position of 5 meters and a velocity of 3 mps.
-      examplePIDSubsyste.SetGoal({5_m, 3_mps});
-
-#### enable() and disable()
-
-The ``enable()`` and ``disable()`` methods enable and disable the motion profiling control of the ``TrapezoidProfileSubsystem``.  When the subsystem is enabled, it will automatically run the control loop and call ``useState()`` periodically.  When it is disabled, no control is performed.
-
-### Full TrapezoidProfileSubsystem Example
-
-What does a ``TrapezoidProfileSubsystem`` look like when used in practice?  The following examples are taking from the ArmbotOffobard example project ([Java](https://github.com/wpilibsuite/allwpilib/tree/main/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/armbotoffboard), [C++](https://github.com/wpilibsuite/allwpilib/tree/main/wpilibcExamples/src/main/cpp/examples/ArmBotOffboard)):
-
-.. tab-set::
-
-   .. tab-item:: Java
-      :sync: Java
-
-      .. remoteliteralinclude:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/v2024.3.2/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/armbotoffboard/subsystems/ArmSubsystem.java
+      .. remoteliteralinclude:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/main/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/drivedistanceoffboard/subsystems/DriveSubsystem.java
```

Comment:
Use a full 40 character commit hash instead of main, as that will keep the RLI from breaking if there's future changes. Inspector will be able to update the commit hash

Reply:
Now that Alpha 2 has been tagged, this can use the tag.